tests/MiSeq.main.nf.test

```diff
@@ -15,15 +15,28 @@ nextflow_pipeline {
         }
 
         then {
+            // stable_name: All files + folders in ${params.outdir}/ with a stable name
+            def stable_name = getAllFilesFromDir(
```

Comment:
Is it possible to refactor these function definitions so they don't have to be redefined for each test?

Reply:
Tbh I've just been copying my homework from sarek here :see_no_evil: I agree it would be nice though, but I'm not sure how to do it, maybe @maxulysse has some better ideas?